index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="wrapper">
+        <header class="main-bar">
+            <div class="left">
+                <p class="time" >9:41</p>
+                <a class="logo" href="#">My Logo</a>
+            </div>
+            <div class="right">
+                <nav class="top-menu">
+                    <svg width="18" height="12" viewBox="0 0 18 12" fill="none" xmlns="http://www.w3.org/2000/svg">
+                        <path fill-rule="evenodd" clip-rule="evenodd"
+                            d="M16.666 0.666748H15.666C15.1137 0.666748 14.666 1.11446 14.666 1.66675V10.3334C14.666 10.8857 15.1137 11.3334 15.666 11.3334H16.666C17.2183 11.3334 17.666 10.8857 17.666 10.3334V1.66675C17.666 1.11446 17.2183 0.666748 16.666 0.666748ZM10.9993 3.00008H11.9993C12.5516 3.00008 12.9993 3.4478 12.9993 4.00008V10.3334C12.9993 10.8857 12.5516 11.3334 11.9993 11.3334H10.9993C10.4471 11.3334 9.99935 10.8857 9.99935 10.3334V4.00008C9.99935 3.4478 10.4471 3.00008 10.9993 3.00008ZM7.33268 5.33341H6.33268C5.7804 5.33341 5.33268 5.78113 5.33268 6.33341V10.3334C5.33268 10.8857 5.7804 11.3334 6.33268 11.3334H7.33268C7.88497 11.3334 8.33268 10.8857 8.33268 10.3334V6.33341C8.33268 5.78113 7.88497 5.33341 7.33268 5.33341ZM2.66602 7.33341H1.66602C1.11373 7.33341 0.666016 7.78113 0.666016 8.33341V10.3334C0.666016 10.8857 1.11373 11.3334 1.66602 11.3334H2.66602C3.2183 11.3334 3.66602 10.8857 3.66602 10.3334V8.33341C3.66602 7.78113 3.2183 7.33341 2.66602 7.33341Z"
```

Comment:
- [ OPTIONAL ] I suggest removing this bar since it's provided by the device. :point_down: 
![Screenshot from 2023-01-20 11-47-54](https://user-images.githubusercontent.com/97834160/213667090-3a1abde0-c51a-45fc-a52b-1874a161d00a.png)